web/controllers/api/docs_controller.ex

```diff
@@ -112,19 +112,27 @@ defmodule HexWeb.API.DocsController do
     version         = to_string(release.version)
     unversioned_key = "docspage/#{package.name}"
     versioned_key   = "docspage/#{package.name}/#{release.version}"
+    pre_release     = length(release.version.pre) > 0
+
+    existing_paths = HexWeb.Store.list_docs_pages("#{name}/")
+    first_release  = length(existing_paths) == 0
```

Comment:
Forgot to push that commit, here it is: https://github.com/hexpm/hex_web/commit/7fc553f9c073a41a3227fa8fe453995c7205c6cb#diff-b0275a36a69e4f0e1a889350f75fc0bbR160.

Your check is even better :+1: